<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GzwsHistoryWindow</class>
 <widget class="QDialog" name="GzwsHistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="mPlot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeFrom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>8</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeTo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>8</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>410</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QToolButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>按时间段查询</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="sliderPosition">
    <number>50</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_lux">
      <property name="text">
       <string>光照度</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_temp">
      <property name="text">
       <string>温度</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_rh">
      <property name="text">
       <string>湿度</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">window/util/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>GzwsHistoryWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
